vmware-event-router/internal/config/v1alpha1/processor.go

```diff
@@ -40,7 +45,11 @@ type ProcessorConfigOpenFaaS struct {
 	Auth *AuthMethod `yaml:"auth,omitempty" json:"auth,omitempty" jsonschema:"description=Authentication configuration for this section"`
 }
 
-// ProcessorConfigEventBridge configures the AWS Event Bridge event processor
```

Comment:
Why did you delete this line? Also your config does not have a comment?

Reply:
Added Back the comment into the code.